server/sonar-server/src/main/java/org/sonar/server/email/ws/SendAction.java

```diff
@@ -74,7 +73,7 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkPermission(GlobalPermissions.SYSTEM_ADMIN);
+    userSession.checkLoggedIn().checkIsRoot();
```

Comment:
We assume that root = system administrator on on-premise installations. So yes, we keep the term `system administrator` in WS descriptions.